programs/drift/src/state/order_params.rs

```diff
@@ -169,11 +174,21 @@ impl OrderParams {
         } else {
             match self.auction_start_price {
                 Some(auction_start_price) => {
+                    let threshold_long = if is_signed_msg {
+                        auction_start_price.safe_mul(999)?.safe_div(1000)?
```

Comment:
to save cu's, can we do `auction_start_price - auction_start_price / 1000`. same for all the other operations